src/grid/CallLayout.ts

```diff
@@ -110,38 +109,33 @@ export function arrangeTiles(
   // use of screen space for n tiles without making those tiles too small or
   // too cropped (having an extreme aspect ratio)
   const gap = width < 800 ? 16 : 20;
-  const tileMinWidth = width < 500 ? 150 : 180;
+  const area = width * minHeight;
+  // Magic numbers that make tiles scale up nicely as the window gets larger
+  const tileArea = Math.pow(Math.sqrt(area) / 8 + 125, 2);
```

Comment:
Is the 8 coming from what we allow as the max number of tiles next to each other?